CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ add_service_files(
   FILES
   FreezeArm.srv
   RelaxArm.srv
+  ProgramNotifier.srv
```

Comment:
I'd like to have all services named with verbs, like functions: CreateProgram.srv